packages/react/src/modules/slack/components/SlackAddChannelInput/styles.css

```diff
@@ -0,0 +1,32 @@
+.rsk-connect-channel {
```

Comment:
Nit: spacing in this file is off, looks like tabs or 4 spaces instead of usual 2

Reply:
very weird my tab spacing got changed in vscode somehow